src/app/chat/views/communities.nim

```diff
@@ -90,11 +103,17 @@ QtObject:
 
       result.add(community)
 
-  proc updateCommunityChat*(self: CommunitiesView, newChat: Chat) =
+  proc updateCommunityChat*(self: CommunitiesView, newChat: Chat, ensureCorrectPositions = false) =
     var community = self.joinedCommunityList.getCommunityById(newChat.communityId)
     if (community.id == ""):
       return
 
+    #if ensureCorrectPositions:
+      # This is to visually put the updated chat in the correct (new)
+      # position so we don't get a funny flicker effect once the position
+      # was updated in the backend.
+      #ensureCorrectChatPositions(community, newChat)
+    #else:
```

Comment:
For the time being I've commented it out to ensure drag and drop works otherwise.